draft-ietf-quic-applicability.md

```diff
@@ -288,10 +288,10 @@ stream frames.
 
 Streams can be unidirectional or bidirectional, and a stream may be initiated
 either by client or server. Only the initiator of a unidirectional stream can
-send data on it. Due to offset encoding limitations, a stream can carry a
-maximum of 2^62-1 bytes in each direction. In the presently unlikely event
-that this limit is reached by an application, the stream can simply be closed
-and replaced with a new one.
+send data on it. Due to encoding limitations on stream offsets and connection
```

Comment:
Splitting this paragraph here might make it flow better.

Reply:
thanks, done